lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1870,3 +1889,49 @@ invariant msg a predicate =
 distance :: (Ord a, Num a) => a -> a -> a
 distance a b =
     if a < b then b - a else a - b
+
+data PoolMetadataSource
+    = None
```

Comment:
Little OCD here, but `None` ~> `FetchNone` for consistency :pray:

Reply:
Done, against my neurologists advice...